apiextensions/v1alpha1/tests/compatibilityrequirements.apiextensions.openshift.io/AAA_ungated.yaml

```
@@ -0,0 +1,1264 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "CompatibilityRequirement"
+crdName: compatibilityrequirements.apiextensions.openshift.io
+tests:
+  onCreate:
+  - name: Should be able to create a minimal CompatibilityRequirement
+    initial: |
+      apiVersion: apiextensions.openshift.io/v1alpha1
+      kind: CompatibilityRequirement
+      metadata:
+        name: test-requirement
+      spec:
+        compatibilitySchema:
+          customResourceDefinition:
+            type: yaml
+            data: |
+              apiVersion: apiextensions.k8s.io/v1
+              kind: CustomResourceDefinition
+              metadata:
+                name: testrequirements.example.com
+              spec:
+                group: example.com
+                names:
+                  kind: TestRequirement
+                  plural: testrequirements
+                scope: Namespaced
+                versions:
+                - name: v1
+                  served: true
+                  storage: true
+                  schema:
+                    openAPIV3Schema:
+                      type: object
+                      properties:
+                        spec:
+                          type: object
+          requiredVersions:
+            defaultSet: StorageOnly
+    expected: |
+      apiVersion: apiextensions.openshift.io/v1alpha1
+      kind: CompatibilityRequirement
+      metadata:
+        name: test-requirement
+      spec:
+        compatibilitySchema:
+          customResourceDefinition:
+            type: yaml
+            data: |
+              apiVersion: apiextensions.k8s.io/v1
+              kind: CustomResourceDefinition
+              metadata:
+                name: testrequirements.example.com
+              spec:
+                group: example.com
+                names:
+                  kind: TestRequirement
+                  plural: testrequirements
+                scope: Namespaced
+                versions:
+                - name: v1
+                  served: true
+                  storage: true
+                  schema:
+                    openAPIV3Schema:
+                      type: object
+                      properties:
+                        spec:
+                          type: object
+          requiredVersions:
+            defaultSet: StorageOnly
+
+  - name: Should not be able to create CompatibilityRequirement without spec
+    initial: |
+      apiVersion: apiextensions.openshift.io/v1alpha1
+      kind: CompatibilityRequirement
+      metadata:
+        name: test-requirement
+    expectedError: "is invalid: [spec: Required value"
```

The error here is picking up the stringification of multiple errors, we normally include the section after the first `[` up to the first `,`, I think that's
```suggestion
    expectedError: "spec: Required value"
```